woocommerce/woocommerce-ios
# [TTPUK] Update UK card configuration for Tap to Pay

Closes: #10872

Merge after: #10886 


## Description


We'll add Tap to Pay on iPhone support for UK WooPayments merchants. This PR enables TTP in the UK, behind a feature flag.

### Changes:
1. Updates the StripeTerminal dependency to the latest in the v2 series, which adds support for Tap to Pay in the UK. 
2. Updates the configuration for the United Kingdom to enable Tap to Pay for UK stores when the feature flag is on.
3. Adds `contactlessLimitAmount` to the configuration, for future messaging around what transactions are eligible for contactless payments, especially relevant to Tap to Pay where that's the only option.

## Testing instructions


Using a build from Xcode, and a UK-based WooPayments store, on an iPhone XS or newer running iOS 16+:
1. Navigate to `Menu > Payments`
2. Observe that `Set up Tap to Pay on iPhone` is visible
3. Run through the flow and check that you can take a test payment using Tap to Pay.

Set the feature flag named `tapToPayOnIPhoneInUK` to false, then repeat the above test – observe that you don't see the Set up Tap to Pay on iPhone option.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/b0c63a23-bef7-44fc-9939-fb8be91779c4




---
